Source/HttpMultipartParser/StreamingMultipartFormDataParser.cs

```diff
@@ -291,8 +291,9 @@ public async Task RunAsync(CancellationToken cancellationToken = default)
         /// <param name="contentDisposition">The content disposition of the multipart data.</param>
         /// <param name="buffer">Some of the data from the file (not neccecarily all of the data).</param>
         /// <param name="bytes">The length of data in buffer.</param>
+        /// <param name="partNo">Serial number of long file part (begins from 0).</param>
         public delegate void FileStreamDelegate(
-            string name, string fileName, string contentType, string contentDisposition, byte[] buffer, int bytes);
+            string name, string fileName, string contentType, string contentDisposition, byte[] buffer, int bytes, int partNo);

```

Comment:
I would rename to `partNumber` in order to be consistent with your other code

```suggestion
            string name, string fileName, string contentType, string contentDisposition, byte[] buffer, int bytes, int partNumber);
```